src/kits/tracker/ContainerWindow.cpp

```diff
@@ -1010,13 +968,41 @@ BContainerWindow::Init(const BMessage* message)
 	SetFlags(Flags() & ~B_NO_WORKSPACE_ACTIVATION);
 }
 
+void
+BContainerWindow::InitLayouts()
+{
+	BLayoutItem* item = 
+		fPoseViewHolder->GroupLayout()->AddView(0, fPoseView->TitleView());
```

Comment:
Ah, Pe has that feature? I had it enabled in other editors, didn't realize Pe had it.